compiler/resolution/wrappers.cpp

```diff
@@ -2213,7 +2218,8 @@ static Expr*      getIndices(PromotionInfo& promotion);
 static Expr*      getIterator(PromotionInfo& promotion);
 
 static bool       haveLeaderAndFollowers(PromotionInfo& promotion,
-                                         CallExpr* call);
+                                         CallExpr* call,
+                                         llvm::SmallVectorImpl<ArgSymbol*>& actualIdxToFormal);
```

Comment:
Why is this not `llvm::SmallVector`?

Reply:
ditto for all other uses

Reply:
Not sure, the surrounding code was like that to start with, so I left it as is. See `promotionWrap`, which is the outermost function here and which uses the `Impl`.